ezplatform/usr/local/share/ez/ez_functions.sh

```diff
@@ -27,8 +27,30 @@ function do_ez_web_server_web_directory_non_writable() {
   fi
 }
 
+function check_ez_database_exists() (
+   set +e
```

Comment:
Are your shell skills good enough to be able to extract the logic from [here](https://github.com/continuouspipe/dockerfiles/blob/395863e83ec2912c36062e4aa285a28e69440736/php/shared/usr/local/share/assets/assets_functions.sh#L96-L114) to a function that you could reuse in this example?

Reply:
Thanks Sam 👍 I should be able to do that, just need to find some time to fit it in 😄